### DISCUSSION TITLE
'was_suite_config.json' is not generated after first first few runs

### DISCUSSION THREAD

**Discussion by tmprabubiz** [WASasquatch/was-node-suite-comfyui#390] [Q&A]
I installed was-node-suite-comfyui in custom_nodes. WHile restarting, I got the following error messages
________________________________________________

J:\ComfyUI\ComfyUI_windows_portable>.\python_embeded\python.exe -s ComfyUI\main.py --windows-standalone-build
** ComfyUI startup time: 2024-04-28 15:13:11.691916
** Platform: Windows
** Python version: 3.11.8 (tags/v3.11.8:db85d51, Feb  6 2024, 22:03:32) [MSC v.1937 64 bit (AMD64)]
** Python executable: J:\ComfyUI\ComfyUI_windows_portable\python_embeded\python.exe
** Log path: J:\ComfyUI\ComfyUI_windows_portable\comfyui.log

Prestartup times for custom nodes:
   0.3 seconds: J:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager

Total VRAM 8192 MB, total RAM 65447 MB
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 2060 SUPER : cudaMallocAsync
VAE dtype: torch.float32
Using pytorch cross attention
Adding extra search path checkpoints C:/Users/REVIVE/Automatic1111/stable-diffusion-webui/models/Stable-diffusion
Adding extra search path configs C:/Users/REVIVE/Automatic1111/stable-diffusion-webui/models/Stable-diffusion
Adding extra search path vae C:/Users/REVIVE/Automatic1111/stable-diffusion-webui/models/VAE
Adding extra search path loras C:/Users/REVIVE/Automatic1111/stable-diffusion-webui/models/Lora
Adding extra search path loras C:/Users/REVIVE/Automatic1111/stable-diffusion-webui/models/LyCORIS
Adding extra search path upscale_models C:/Users/REVIVE/Automatic1111/stable-diffusion-webui/models/ESRGAN
Adding extra search path upscale_models C:/Users/REVIVE/Automatic1111/stable-diffusion-webui/models/RealESRGAN
Adding extra search path upscale_models C:/Users/REVIVE/Automatic1111/stable-diffusion-webui/models/SwinIR
Adding extra search path embeddings C:/Users/REVIVE/Automatic1111/stable-diffusion-webui/embeddings
Adding extra search path hypernetworks C:/Users/REVIVE/Automatic1111/stable-diffusion-webui/models/hypernetworks
Adding extra search path clip_vision C:/Users/REVIVE/Automatic1111/stable-diffusion-webui/models/clip_vision/
Adding extra search path controlnet C:/Users/REVIVE/Automatic1111/stable-diffusion-webui/models/ControlNet
Adding extra search path ipadapter C:/Users/REVIVE/Automatic1111/stable-diffusion-webui/models/ControlNet
Traceback (most recent call last):
  File "J:\ComfyUI\ComfyUI_windows_portable\ComfyUI\nodes.py", line 1864, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 936, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1073, in get_code
  File "<frozen importlib._bootstrap_external>", line 1130, in get_data
FileNotFoundError: [Errno 2] No such file or directory: 'J:\\ComfyUI\\ComfyUI_windows_portable\\ComfyUI\\custom_nodes\\ComfyUI-HyperSDXL1StepUnetScheduler\\__init__.py'

Cannot import J:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-HyperSDXL1StepUnetScheduler module for custom nodes: [Errno 2] No such file or directory: 'J:\\ComfyUI\\ComfyUI_windows_portable\\ComfyUI\\custom_nodes\\ComfyUI-HyperSDXL1StepUnetScheduler\\__init__.py'
### ComfyUI Revision: 2128 [258dbc06] | Released on '2024-04-14'
Efficiency Nodes Warning: Failed to import python package 'simpleeval'; related nodes disabled.

Traceback (most recent call last):
  File "J:\ComfyUI\ComfyUI_windows_portable\ComfyUI\nodes.py", line 1864, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "J:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\was-node-suite-comfyui\__init__.py", line 1, in <module>
    from .WAS_Node_Suite import NODE_CLASS_MAPPINGS
  File "J:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\was-node-suite-comfyui\WAS_Node_Suite.py", line 37, in <module>
    from numba import jit
ModuleNotFoundError: No module named 'numba'

Cannot import J:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\was-node-suite-comfyui module for custom nodes: No module named 'numba'

Import times for custom nodes:
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/alter-list.json
   0.0 seconds: J:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\websocket_image_save.py
   0.0 seconds: J:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-tcd-scheduler
   0.0 seconds (IMPORT FAILED): J:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-HyperSDXL1StepUnetScheduler
   0.0 seconds (IMPORT FAILED): J:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\was-node-suite-comfyui
   0.0 seconds: J:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\efficiency-nodes-comfyui-main
   0.2 seconds: J:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager

[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/model-list.jsonStarting server

To see the GUI go to: http://127.0.0.1:8188
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
FETCH DATA from: J:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\extension-node-map.json
___________________________________________________________________________

Also 'was_suite_config.json' is not generated inside the folder 'J:\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\was-node-suite-comfyui'. Please guide

**Comment by noarche** [comment]
`python_embeded\python -m pip install numba`
